app/models/booking.rb

```diff
@@ -1,5 +1,16 @@
 class Booking < ApplicationRecord
   belongs_to :animal
   belongs_to :user
-  validates :date, :start_time, :duration, :total_fee, :animal, :user, presence: true
+  validates :start_date, :end_date, :total_fee, :animal, :user, presence: true
```

Comment:
do we need a validates presence true for total_fee? we will calculate that one based on the days and rate, right? It's not a user input that we need to make sure is there

Reply:
I added it as a field temporarily just to be able to test that the model and controller were working correctly, it was only a placeholder. I've removed it now.